src/SprykerEco/Zed/Computop/Persistence/ComputopEntityManager.php

```diff
@@ -46,8 +50,8 @@ public function updatePaymentComputopOrderItemPaymentConfirmation(
         $paymentComputopOrderItemEntities = $this->getFactory()
             ->createPaymentComputopOrderItemQuery()
             ->useSpyPaymentComputopQuery()
-                ->filterByTransId($computopNotificationTransfer->getTransId())
-                ->filterByPayId($computopNotificationTransfer->getPayId())
+            ->filterByTransId($computopNotificationTransfer->getTransId())
```

Comment:
Usually, we use additional indentation when filtering/joining something using `use...Query()`.